admin/src/components/LinksTransactionListFormular.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="links-transaction-list-formular">
```

Comment:
I suggest `TransactionLinkList` as name for this component

Reply:
name geändert